C++/Toolbox/Option/Option.h

```diff
@@ -0,0 +1,127 @@
+#ifndef _OPTION_H_
+#define _OPTION_H_
+
+#include <functional>
+#include <memory>
+#include <type_traits>
+
+namespace Toolbox {
```

Comment:
I would switch to using the Toolbox::Option namespace convention.

Reply:
done